CMakeLists.txt

```diff
@@ -96,6 +97,7 @@ option(WITH_EXTENDED_FILTERING "Set to OFF to build without extended filtering.
 
 option(WITH_DLT_DAEMON_VSOCK_IPC "Set to ON to enable VSOCK support in daemon"                                   OFF)
 option(WITH_DLT_LIB_VSOCK_IPC    "Set to ON to enable VSOCK support in library (DLT_IPC is not used in library)" OFF)
+option(WITH_DLT_LOG_LEVEL_APP_CONFIG "Set to ON to allow configuration of log levels per application" OFF)
```

Comment:
it is correct with WITH_LOG_LEVEL_APP_CONFIG

Reply:
I'd rather keep WITH_DLT_LOG_LEVEL_APP_CONFIG because most other options do contains _DLT_ as well.

Reply:
replaced `WITH_LOG_LEVEL_APP_CONFIG` with `WITH_DLT_LOG_LEVEL_APP_CONFIG` and added compile option in doc/dlt_build_options.md